docs/reduction_toolbox.rst

```diff
@@ -428,6 +432,112 @@ which should be removed rather than propagated:
     plt.show()
 
 
+Working with multi-extension FITS image files
+---------------------------------------------
+
+Multi-extension FITS (MEF) image files cannot be processed natively in ``ccdproc``. The example below illustrates how to `~ccdproc.flat_correct` all of the extensions in a MEF and write out the calibrated file as a MEF. Applying other reduction steps would be similar.
+
+The example begins by creating small MEF "science" image and a sample flat:
+
+    >>> import numpy as np
```

Comment:
Since we're not actually showing output of any operation can't we drop the `>>>` and `...`?

Reply:
I'd like to keep it as a doctest, though, to ensure that the code actually runs.

Reply:
Ah, the code isn't doc-tested if it doesn't have the `>>>`?